source/Running.md

```diff
@@ -0,0 +1,379 @@
+# Running
+
+It is recommended to `change default ProxySQL credentials
+<default-credentials>`{.interpreted-text role="ref"} before running
```

Comment:
{.interpreted-text role="ref"}  should be removed or substituted with the link

Reply:
fixed